docs/pages/routing/introduction.mdx

```diff
@@ -23,6 +24,8 @@ It brings the best file-system routing concepts from the web to a universal appl
 - **Universal**: iOS, Android, and web share a unified navigation structure, with the ability to drop-down to platform-specific APIs at the route level.
 - **Discoverable**: Expo Router enables build-time static rendering on web, and universal linking to native. Meaning your app content can be indexed by search engines.
 
+<Video file="expo-router/routing.mp4" loop={false} />
```

Comment:
The video is outdated (if from expo/router docs), uses the old layout convention.

Reply:
Thanks for catching that! Removed the video from the doc and the static file.